### PR TITLE
Bump version: 0.37.0-beta → 0.37.0b2

### PR DESCRIPTION
**Pull request by konradkonrad**
### What this PR does
Bump the PyPi version of `raiden-contracts` to `0.37.0b2`.

### Why I'm making this PR
This will allow us to upload a new beta release to PyPi

### What's tricky about this PR (if any)

I used `bump2version --config-file .bumpversion.cfg --new-version '0.37.0-beta2' 0.37.0-beta2`, but still had to
manually adjust the `VERSION` constant in `setup.py`. Maybe there are other places, that I missed?

----

Any reviewer can check these:

* [ ] If the PR is fixing a bug or adding a feature, add an entry to CHANGELOG.md.
* [ ] If the PR changed a Solidity source, run `make compile_contracts` and add the resulting `raiden_contracts/data/contracts.json` in the PR.
* [ ] If the PR is changing documentation only, add `[skip ci]` in the commit message so Travis does not waste time.
    * [ ] But, if the PR changes comments in a Solidity source, do not add `[skip ci]` and let Travis check the hash of the source.
* [ ] In Python, use keyword arguments
* [ ] Squash unnecessary commits
* [ ] Comment commits
* [ ] Follow naming conventions
    * `solidityFunction`
    * `_solidity_argument`
    * `solidity_variable`
    * `python_variable`
    * `PYTHON_CONSTANT`
* [ ] Follow the Signature Convention in [CONTRIBUTING.md](./CONTRIBUTING.md)
* For each new contract
    * [ ] The deployment script deploys the new contract.
    * [ ] `etherscan_verify.py` runs on the new contract.
* Bookkeep
    * [ ] The gas cost of new functions are stored in `gas.json`.
* Solidity specific conventions
    * [ ] Document arguments of functions in natspec
    * [ ] Care reentrancy problems
    * if the PR adds or removes `require()` or `assert()`
        * [ ] add an entry in Changelog
        * [ ] open an issue in the client, light client, service repos so the change is reflected there
        * Just adding a message in `require()` doesn't require these steps.
* [ ] When you catch a require() failure in Solidity, look for a specific error message like `pytest.raises(TransactionFailed, match="error message"):`

And before "merge" all checkboxes have to be checked.  If you find redundant points, remove them.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from setuptools.command.build_py import build_py
 
 DESCRIPTION = "Raiden contracts library and utilities"
-VERSION = "0.37.0-beta"
+VERSION = "0.37.0"
```
Comment (Dominik1999) [inline setup.py:14]: why is this not `-beta2`?

**Comment by ulope** [comment]
Nitpick: the "proper" / canonical spelling would be `0.37.0b2`

File: setup.py (hunk 1 above)
Comment (konradkonrad) [reply setup.py:14]: wait....that's exactly what I tried to fix here. give me a second!

**Comment by konradkonrad** [comment]
> Nitpick: the "proper" / canonical spelling would be `0.37.0b2`

The normalization actually happens automatically. I.e. `"0.37.0-beta"` earlier always folded into `0.37.0b0` (see e.g. https://travis-ci.org/github/raiden-network/raiden-contracts/jobs/672460279#L6103 )

**Comment by ulope** [comment]
Yeah but to avoid problems with tools that don't know about PEP440 normalization I think it would make sense to use the canonical version everywhere.

**Comment by konradkonrad** [comment]
> Yeah but to avoid problems with tools that don't know about PEP440 normalization I think it would make sense to use the canonical version everywhere.

I see. I tried to stay consistent with the previous scheme. But I'll change it :)

**Pull request merged**